src/Handlers.jl

```diff
@@ -254,7 +254,7 @@ function Base.insert!(node::Node, leaf, segments, i)
     end
 end
 
-function Base.match(node::Node, params, method, segments, i)
+function Base.match(node::Node, method, segments, i)
```

Comment:
Drive by comment, but if this is just an internal function, maybe it shouldn't overload Base.match?

Reply:
Yeah, that's a good thought; same with `insert!`.